vignettes/getting-started.Rmd

```diff
@@ -50,22 +50,23 @@ The _first time_ you use `rbabylon` on a system or disk, you must first install
 
 To reiterate, there are two relevant paths for this section.
 
-* `BBI_EXE_PATH` -- the path the `babylon` executable file on your system.
+* `BBI_EXE_PATH` -- the path to the `babylon` executable file on your system.
 * `MODEL_DIR` -- the root modeling directory where you will be running models with `rbabylon`
 
 ```{r define dirs}
-BBI_EXE_PATH <- "/data/apps/bbi" # this should be an absolute path
+BBI_EXE_PATH <- read_bbi_path()
 MODEL_DIR <- "../inst/nonmem"    # this should be relative to your working directory
 ```
 
 ### Installing babylon
 
 You can install the `babylon` binary with:
-```
+
+```{r, eval = FALSE}
 rbabylon::use_bbi()
 ```
 
-This will install to the `/data/apps/` folder by default, which matches the `BBI_EXE_PATH` you just defined (and will use soon). If you have changed that, you can pass your custom path to the _directory_ where you want to put bbi like so `use_bbi("/some/other/dir")`. In that case, you would want to set `BBI_EXE_PATH = "/some/other/dir/bbi"`.
+This will install to the `/data/apps/` folder by default, which will usually match the `BBI_EXE_PATH` you just defined (it will not match only if the `BABYLON_EXE_PATH` environment variable is deliberately set to a different path). If you have changed that, you can pass your custom path to the _directory_ where you want to put bbi like so `use_bbi("/some/other/dir")`. In that case, you would want to set `BBI_EXE_PATH = "/some/other/dir/bbi"`.
```

Comment:
yea, unless you strongly disagree, I think we should revert this back to the original. No one but developers and _maybe_ very advanced users should be using this environment variable. This is a "Getting Started" vignette. Let's try to keep it as simple as possible. Setting this path up already seems to confuse some users.